### DISCUSSION TITLE
Acesso Externo

### DISCUSSION THREAD

**Discussion by Betauum2022** [RamonSilva20/mapos#1973] [General]
Alguem pode me orientar como que eu faço para conseguir acessar externamente, tenho ip valido porem não estou conseguindo acessar, o mais estranho de tudo que consigo acessar o cockpit com a porta 9090. Lembrando que estou usando ele no ubuntu server.

**Comment by evertonterra** [comment]
Eu também não conseguia acessar de fora da rede interna nem mesmo pela vpn, não aceitava acessar se não fosse com o IP interno , apenas comentei a linha e passou a acessar , faça esse teste espero que veja e que isso lhe ajude 🙏
//$config['base_url'] = 'http://192.168.0.116/mapos
